lucetc/src/signature.rs

```diff
@@ -74,6 +75,12 @@ pub fn verify_source_code(
 }
 
 // Sign the compiled code
+#[cfg(feature = "signature_checking")]
 pub fn sign_module<P: AsRef<Path>>(path: P, sk: &SecretKey) -> Result<(), Error> {
     ModuleSignature::sign(path, sk).map_err(|e| e.into())
 }
+
+#[cfg(not(feature = "signature_checking"))]
+pub fn sign_module<P: AsRef<Path>>(_path: P, _sk: &SecretKey) -> Result<(), Error> {
+    Ok(())
```

Comment:
If that public function is called but the signature feature is not available, how about throwing an error instead?

Reply:
Yup, makes sense. Will fix... Looks like I forgot to run the cargo fmt as well on the last submission as well.